content/docs/reference/autocert.mdx

```diff
@@ -15,47 +17,42 @@ import TabItem from '@theme/TabItem';
 
-## Autocert {#autocert}
+:::note Autocert support
+
+The suite of Autocert settings listed below are supported only in [Pomerium Core](/docs/core) configurations. Here's what you should know about ACME support in other Pomerium products:
 
-Turning on Autocert allows Pomerium to automatically retrieve, manage, and renew public facing TLS Certificates from [Let's Encrypt][letsencrypt], which includes managed routes and the Authenticate Service.
+- Pomerium Enterprise users should note that Autocert cannot be configured in the [Enterprise Console](/docs/enterprise#enterprise-console) or the Pomerium Enterprise [configuration file](/docs/enterprise/configure). Autocert is a bootstrap setting, meaning it must be configured in Pomerium Core (either in a configuration file or as an environment variable) before running Pomerium Enterprise.
 
-You must use [Autocert Directory](/docs/reference/autocert) with Autocert to store and share certificate data between services. (Note that Autocert also provides [OCSP stapling](https://en.wikipedia.org/wiki/OCSP_stapling).)
+- Kubernetes users should refer to the [Cert-Manager integration](/docs/k8s/ingress#cert-manager-integration) section for more information on configuring automated certificate management in the Pomerium Ingress Controller.
 
-The Autocert setting can be useful in situations where you may not have Pomerium behind a TLS terminating ingress or proxy that is already handling your public certificates on your behalf.
+- Pomerium Zero users should note that Pomerium automatically manages certificates for [starter domains](/docs/concepts/clusters#starter-domain) and [custom domains](/docs/capabilities/custom-domains).
 
-Autocert will incorporate certificates available in the system trust store and certificates set manually in the Pomerium configuration, and they will take precedence over generated certificates when applicable to configured routes.
+:::
+
+## Autocert {#autocert}
 
-Autocert will attempt `HTTP-01`and `TLS-ALPN-01` challenges. It does not support `DNS-01` challenges, required to generate wildcard certificates.
+The **Autocert** setting enables Pomerium to automatically retrieve, manage, and renew public-facing TLS certificates from [Let's Encrypt][letsencrypt] to secure Pomerium-managed route domains and the authenticate service (Autocert is not supported with the [Hosted Authenticate Service](/docs/capabilities/hosted-authenticate-service)).
 
-Kubernetes users should not use Autocert. See the [cert-manager's guide](https://cert-manager.io/docs/tutorials/acme/pomerium-ingress/) for more information.
+Autocert can incorporate also certificates available in the system trust store and certificates set manually in the Pomerium configuration. These certificates will take precedence over generated certificates when applicable to configured routes.
```

Comment:
I think I mentioned this in Slack, but I don't know that this is accurate. I think it would be unusual to configure a server application certificate via the system trust store.

Reply:
I updated this section to remove the system trust store detail.